### DISCUSSION TITLE
Display performance for week and month

### DISCUSSION THREAD

**Discussion by raider-com** [ghostfolio/ghostfolio#1386] [Ideas]
Hi:
Not sure if I'm blind or the feature doesn't exist.  When looking at the overview section, there are options for today, one year, 5 years and max.  Is there anyway to view 1 week and 1 month?
Please forgive me if this isn't the right place to ask.  Had a hard time finding another forum for this question.
Thanks
J

**Comment by dtslvr** [comment]
Hi @raider-com

Thank you for your suggestion. It is currently not possible to show the performance for 1 `Week` / `Month`. `Today`, `YTD`, `1Y`, `5Y` and `MAX` are supported.

Are you more interested in seeing the performance of the last month or an overview of the performances for each month (Jan to Dec)?

**Comment by raider-com** [comment]
I'm trying to replace a series of spreadsheets that I have.
In an ideal state, I'd like to have 1 week to today, 1 month ago to today, and an overview of month performances month over month (say the end of the month to end of month) (access via a different page, since it would be a bit more complicated), but the first 2 would be more important to me.
Not sure if that makes sense or not.
Thanks
-J

**Comment by meuter** [comment]
Hi,

> Are you more interested in seeing the performance of the last month or an overview of the performances for each month (Jan to Dec)?

I would say both ;-)

Having month-to-date (since the first of the month) and week-to-date (since the beginning of the week) in the date-range panel would be great to have indeed. More generally, I would love to be able to specify custom range in this panel (start+end date).  This would allow for instance to view yearly performance for a given year. By date-range panel, I mean this one in the upper right corner:
![image](https://github.com/ghostfolio/ghostfolio/assets/843867/74e49133-d40f-4108-b0e1-509e3a7f9c49)

As for a general overview, a monthly/yearly heat map would be ideal IMHO. Something like this (from Portfolio Performance)
![etf-returns-until-2023-portfolio-performance](https://github.com/ghostfolio/ghostfolio/assets/843867/0c177089-cfca-4b42-aa57-d5314afd8ff2)

Thanks

-Cédric

**Comment by dtslvr** [comment]
@meuter I have created a task as a starting point: https://github.com/ghostfolio/ghostfolio/issues/2900

**Comment by tmeekes** [comment]
The MTD and WTD options are valuable, but I'd say there's still additional value in "1 month" and "1 week" options as well, for displaying those rolling time periods instead.
